src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = days * 40;
```

Comment:
You use the expression: (days * 40) in other places, so let's create a separate variable for this.